lib/vstd/StringUtils.cpp

```diff
@@ -0,0 +1,13 @@
+#include <vstd/StringUtils.h>
```

Comment:
`#include "StdInc.h"` must be first line

Reply:
fixed.